tests/admin_views/tests.py

```diff
@@ -4453,6 +4453,34 @@ def test_inline_uuid_pk_delete_with_popup(self):
         self.assertEqual(select.first_selected_option.text, '---------')
         self.assertEqual(select.first_selected_option.get_attribute('value'), '')
 
+    def test_list_editable_raw_id_fields(self):
+        class RelatedWithUUIDPKModelAdmin(ModelAdmin):
+            list_display = ['pk', 'parent']
+            list_editable = ['parent']
+            raw_id_fields = ['parent']
+
+        site.unregister(RelatedWithUUIDPKModel)
+        site.register(RelatedWithUUIDPKModel, RelatedWithUUIDPKModelAdmin)
```

Comment:
This test passes in isolation but not when run with the other tests. There is some caching going on that prevents unregister/register from working as I'd like. Perhaps someone can spot the reason why or suggest some other workaround. Registering this admin for all tests causes failures because it changes the UI in ways that affect them.